devtools/prettier.py

```diff
@@ -239,6 +246,26 @@ def _format_dataclass(self, value: 'Any', _: str, indent_current: int, indent_ne
             self._stream.write(',\n')
         self._stream.write(indent_current * self._c + ')')
 
+    def _format_sqlalchemy_class(self, value: 'Any', _: str, indent_current: int, indent_new: int):
+        import json
+
+        fields = {}
+        for field in [x for x in dir(value) if not x.startswith('_') and x != 'metadata']:
+            data = value.__getattribute__(field)
```

Comment:
If there's a good reason for this instead of `getattr(value, field)` it should be commented.